src/ipc/show-save-dialog.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  IpcMain,
+  IpcMainInvokeEvent,
+  dialog,
+  SaveDialogOptions,
+} from 'electron';
+
+export const channel = 'show-save-dialog';
```

Comment:
nit: style guide calls for `CONSTANT_CASE` top-level constants

Reply:
For whatever other idiomatic reason, all of the channel constants across all the IPC files are `channel`.